src/Broker/Broker.c

```diff
@@ -80,6 +81,10 @@ Broker_initialize_packetizers_functions()
     packetizers_functions[PACKETIZER_THIN].init = &ThinPacketizer_init;
     packetizers_functions[PACKETIZER_THIN].packetize = &ThinPacketizer_packetize;
     packetizers_functions[PACKETIZER_THIN].depacketize = &ThinPacketizer_depacketize;
+    packetizers_functions[PACKETIZER_DEVICE_PROVIDED].headerSize = THIN_SPACE_PACKET_PRIMARY_HEADER_SIZE;
```

Comment:
Will this be the same for all device provided packetizers?

Reply:
Done

Reply:
THIN_SPACE_PACKET_PRIMARY_HEADER_SIZE set as default, It is changed later in DeviceProvidedPacketizer

Reply:
Set to 0